go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/antham/chyle v1.6.0
 	github.com/aws/aws-sdk-go v1.24.0
 	github.com/awslabs/goformation v0.0.0-20190320125420-ac0a17860cf1
-	github.com/banzaicloud/bank-vaults v0.0.0-20190508130850-5673d28c46bd
+	github.com/banzaicloud/bank-vaults v0.0.0-00010101000000-000000000000
```

Comment:
I guess that's ok since we still have the explicit replace, right?

Reply:
Yeah, I don't know why this decided to happen, but it did. The replace does seem to do the trick.